pkg/smartcontract/manifest/manifest.go

```diff
@@ -155,6 +158,7 @@ func (m *Manifest) ToStackItem() (stackitem.Item, error) {
 	return stackitem.NewStruct([]stackitem.Item{
 		stackitem.Make(m.Name),
 		stackitem.Make(groups),
+		stackitem.NewMap(),
```

Comment:
We may need to add a comment to neo-project/neo#2345 about the map representation of `features` field. `stackitem.Map` requires preserving the order of fields, but JSON specification can't provide it. Thus, we need `Manifest.Features` either to be `stackitem.ByteArray` (the same solution was applied in https://github.com/neo-project/neo/pull/2254 to Manifest.Extra) or to have strictly-defined structure.

Reply:
However, this can wait until neo will support non-empty `features`.